includes/admin/reporting/reports.php

```diff
@@ -2384,9 +2261,9 @@ function edd_register_customer_report( $reports ) {
 			'priority'  => 40,
 			'endpoints' => array(
 				'tiles'  => array(
-					'lifetime_value_of_customer',
+					'average_revenue_per_customer',
 					'average_number_of_orders_per_customer',
-					'average_customer_value',
+					'average_customer_revenue'
```

Comment:
I don't think a tile exists for either `average_customer_value` or `average_customer_revenue`. I'm seeing three tiles registered but only two display ("Average Revenue Per Customer" and "Average Number of Orders").

Additionally, the average number of orders seems to be incorrect. My top five customers is showing 52 for the highest number of sales for a customer, but the average number of orders is 222.